bool enable
bool disable
bool manual
bool automated
bool initpose
